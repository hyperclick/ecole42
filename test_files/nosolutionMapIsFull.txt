7.o
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
