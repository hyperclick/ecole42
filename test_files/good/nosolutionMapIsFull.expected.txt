oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
oooooooooooooooooooooooooooo
